src/app/containers/SocialEmbed/sourceHelpers.js

```diff
@@ -0,0 +1,30 @@
+const PROVIDERS = {
+  TWITTER: 'twitter',
+  UNKNOWN: 'unknown',
+};
+
+/**
+ * Determine the name of a supported provider from a social embed source URL.
+ * @param {string} source The source URL of a social embed.
+ * @returns {string} Provider name.
+ */
+export const getProviderFromSource = source => {
+  if (source.match(/^https:\/\/twitter\.com/)) {
```

Comment:
I assume so, but is this URL always provided in this form?  eg, always `https` with no subdomain like `www`?

Reply:
Yes, for Tweets, we understand this is the only URL form. If this changes, we can update the matching pattern. The worst that will happen is a fallback is rendered as opposed to a full Tweet.